src/network/keys.js

```diff
@@ -1,6 +1,4 @@
-export const keys = {
-    USERS: "users",
-    USER_POSTS: "user-posts",
-    POST_COMMENTS: "post-comments",
-};
-
+export const USERS_KEY = 'users';
+export const USER_POST_KEY = 'user-post';
+export const USER_POSTS_KEY = 'user-posts';
+export const POST_COMMENTS_KEY = 'post-comments';
```

Comment:
Do we consume these keys at multiple instances or have any dependency on them? What would be the point of storing them in const?

Reply:
These keys are used to identify different queries in React Query. While these keys are mainly associated with React Query, we do not have explicit dependencies on them outside of the queries themselves. We've chosen to store them as constants for code re-usability and maintainability.